RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -4527,3 +4527,26 @@ boolean auto_burnMP(int mpToBurn)
 	return startingMP != my_mp();
 }
 
+boolean can_read_skillbook(item it) {
+  if (it == $item[spinal-fluid-covered emotion chip] && in_robot()) {
+    return true;
+  }
+  if (is_boris() || is_jarlsberg() || is_pete() || in_zombieSlayer() || isActuallyEd() || in_awol() || in_gnoob() || in_darkGyffte() || in_robot() || in_wereprof()) {
```

Comment:
honestly this breaks every time there's a new avatar path, it should probably instead check that you're a standard class and return true, otherwise return false by default. I'll approve as is but worth thinking about

Reply:
unfortunately AoSOL /can/ read skillbooks (Pig Skinner / Cheese Wizard / Jazz Agent). I feel like that might have been a mistake.

as I missed plumber I think this is sensible.